samples/bridge/service-control/Bridge_1/Endpoint/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using NServiceBus;
+
+class Program
+{
+    static async Task Main()
+    {
+        Console.Title = "Samples.Bridge.Endpoint";
+        const string letters = "ABCDEFGHIJKLMNOPQRSTUVXYZ";
+        var random = new Random();
+        var endpointConfiguration = new EndpointConfiguration(
+            "Samples.Bridge.Endpoint");
+
+        endpointConfiguration.UseTransport(new LearningTransport
+        {
+            StorageDirectory = $"{LearningTransportInfrastructure.FindStoragePath()}2"
```

Comment:
Perhaps use a better name than "2"?

Reply:
I used this because it has been used in the simple bridge sample -- to be consistent.